extensions/amp-mathml/1.0/base-element.js

```diff
@@ -10,7 +10,7 @@ BaseElement['Component'] = BentoMathml;
 
 /** @override */
 BaseElement['props'] = {
-  'inline': {attr: 'inline', type: 'boolean', default: false},
```

Comment:
isn't the default boolean value `undefined`? https://github.com/ampproject/amphtml/blob/main/src/core/dom/index.js#L480

`parseBooleanAttribute() === undefined`

Reply:
you're right